Library/Formula/cvc4.rb

```diff
@@ -0,0 +1,40 @@
+class Cvc4 < Formula
+  homepage "http://cvc4.cs.nyu.edu/"
+  url "http://cvc4.cs.nyu.edu/builds/src/cvc4-1.4.tar.gz"
+
+  devel do
+    url "http://cvc4.cs.nyu.edu/builds/src/unstable/latest-unstable.tar.gz"
```

Comment:
Would it be better as `head` instead, then, with less of an expectation of stability? They only keep the most recent `n` snapshots in that directory, so a link to a particular date would go stale pretty quickly.